common/src/main/java/com/microsoft/identity/common/internal/controllers/ApiDispatcher.java

```diff
@@ -56,6 +57,14 @@ public static void beginInteractive(final InteractiveTokenCommand command) {
                 "Beginning interactive request"
         );
         synchronized (sLock) {
+
+            if(!sInteractiveExecutor.isTerminated()){
```

Comment:
Are we calling `shutdown()` or `shutdownNow()` anywhere?

Reply:
Actually this if loop is not needed, was part of my older change, missed to remove, we just have to send a broadcast and if there is an active `AuthorizationActivity` to will finish itself.

Updated it. Thanks Brian